themes/bootstrap3/js/common.js

```diff
@@ -408,9 +408,11 @@ var VuFind = (function VuFind() {
       link.addEventListener('click', function toggleQRCode() {
         var holder = this.nextElementSibling;
         if (holder.querySelectorAll('img').length === 0) {
-          // We need to insert the QRCode image
-          var template = holder.querySelector('.qrCodeImgTag').innerHTML;
-          holder.innerHTML = template;
+          // Replace the QRCode template with the image:
```

Comment:
The old code replaced `<li>` with img, this replaces just the content of the list item.